shell/platform/darwin/common/buffer_conversions.mm

```diff
@@ -7,22 +7,22 @@
 #include "flutter/fml/macros.h"
 #include "flutter/fml/platform/darwin/scoped_nsobject.h"
 
+static_assert(__has_feature(objc_arc), "ARC must be enabled.");
```

Comment:
Since this isn't part of the framework and doesn't have dependencies on it, we do things manually.